src/index.js

```diff
@@ -509,6 +509,38 @@ export default function ({ types: t }) {
           extendReactElementWithRef(path);
         }
       },
+      // RN screen selector support
+      JSXOpeningElement: function JSXOpeningElement(path, state) {
+        const filename = state.file.opts.filename;
+        const isReactNavigationFile = filename.includes('node_modules/@react-navigation');
+        if (!isReactNavigationFile) {
+          return;
+        }
+
+        // rewrite all `<MaybeScreen />` components.
+        const isMaybeScreenView = path.node.name.name === 'MaybeScreen';
```

Comment:
is `path.node.name.name` guaranteed to always resolve?

Reply:
Yes, `path.node.name` is always `JSXIdentifier | JSXMemberExpression | JSXNamespacedName`, which all have `name` property. 

https://babeljs.io/docs/babel-types#jsxopeningelement

I'll add an early return to check for `isJSXIdentifier` to be more specific though.